codegen/datamodelgen.py

```diff
@@ -118,6 +121,9 @@ def _populate_static_info(self):
         run_solver_mode = any(
             info.mode == "solver" for _, info in self._static_info.items()
         )
+        run_icing_mode = "PYFLUENT_FLUENT_ROOT" in os.environ.keys() and any(
```

Comment:
How can we know that we will be connecting to a v231 ?  Right now the hardcoded state is AWP_ROOT222 I think,
so we can assume that using PYFLUENT_FLUENT_ROOT is "probably" v231.

Reply:
It's not hardcoded to 222 any more.  One can call set_fluent_version(23.1).

Reply:
set_ansys_version() yes

but for codegen, either we use an environment variable or add a command line argument.
maybe

python codegen/allapigen.py --ansysVersion=23.1

Reply:
You mean if you do nothing then when Fluent is launched 22.2 is launched by default, but you want to be able to run all this and have launch_fluent use 23.1 instead?

Reply:
@seanpearsonuk @dnwillia-work added optional command line arguments to codegen, for convenience.